pep/app/pods/search/route.ts

```diff
@@ -116,7 +118,7 @@ export default class Search extends PageNav(Route) {
         controller.currentViewedCount = controller.viewedCount;
         controller.currentViewedPeriod = controller.viewedPeriod;
         controller.currentSearchTerms = isEmpty(controller.searchTerms)
-            ? [{ type: 'everywhere', term: '' }]
+            ? cloneDeep(SEARCH_DEFAULT_TERMS)
```

Comment:
My minor gripe with this is I really don't love the name `cloneDeep` because its not clear why or what its doing this for. 

Thoughts about creating a `getInstance` helper that calls `cloneDeep` in it?

Reply:
i think `cloneDeep` more accurately describes what its actually doing tho, since its literally deeply/recursively cloning the given object, its not getting an instance of something via like `new obj()` or anything.

I could just rename the import to like `copy` or `copyObject()` or something if you preferred? As well as add comments here as to why its needed.

Reply:
That sounds good